### PR TITLE
Update CHANGELOG to graphql api names for tis

### PR DESCRIPTION
**Pull request by jinjoolee07**
Fixes: #805

### CODE REVIEW

**Comment by kimhanbeom** [comment]
- This part should be changed to `requestFromPeer` as it is about the GraphQL API.
https://github.com/aicers/giganto/blob/3fd2c26948ec5b6d2366adf273f359fba98725ff/CHANGELOG.md?plain=1#L22

- `total_count` (GraphQL) query -> `totalCount` GraphQL API
https://github.com/aicers/giganto/blob/3fd2c26948ec5b6d2366adf273f359fba98725ff/CHANGELOG.md?plain=1#L32
https://github.com/aicers/giganto/blob/3fd2c26948ec5b6d2366adf273f359fba98725ff/CHANGELOG.md?plain=1#L49

**Comment by jinjoolee07** [comment]
> * This part should be changed to `requestFromPeer` as it is about the GraphQL API.
>   https://github.com/aicers/giganto/blob/3fd2c26948ec5b6d2366adf273f359fba98725ff/CHANGELOG.md?plain=1#L22
> * `total_count` (GraphQL) query -> `totalCount` GraphQL API
>   https://github.com/aicers/giganto/blob/3fd2c26948ec5b6d2366adf273f359fba98725ff/CHANGELOG.md?plain=1#L32
>   
>   https://github.com/aicers/giganto/blob/3fd2c26948ec5b6d2366adf273f359fba98725ff/CHANGELOG.md?plain=1#L49

Modification Completed

**Review by kimhanbeom — APPROVED** [review]

**Comment by sophie-cluml** [comment]
@jinjoolee07 Most of the content in this branch should match the main branch, only some tis branch stuff should be different from the main branch. Could you please confirm that the parts that should be the same as the main branch are the same?

**Comment by jinjoolee07** [comment]
> @jinjoolee07 Most of the content in this branch should match the main branch, only some tis branch stuff should be different from the main branch. Could you please confirm that the parts that should be the same as the main branch are the same?

Here are the lines that I have aligned with the main branch to ensure they match:

- Added GraphQL API `ping` and mutation `stop`, `reboot`, `shutdown`.
- Modify `retain_periodically` function to run as a separate thread.
- Modify the `sources` GraphQL API to return results that also include sources from its peers.
- Modify the related GraphQL APIs as the db key of `Netflow5`/`Netflow9`/`SecuLog` is changed to source.(`netflow5_raw_events`/`netflow9_raw_events`/ `secu_log_raw_events`)
- Added `SmtpRawEvent` to the return value union of `networkRawEvents` GraphQL API.
- Modify `export` GraphQL API to set the extension of the extract file according to the export type.
- Fix the part of the `export` query in the GraphQL API about validating filters for protocols.
- Set the `AckTransmissionCount` by reading from the config file.
- Added `setAckTransmissionCount` GraphQL API to set the ack transmission count. This query changes the `AckTransmissionCount` used in ingest and `ack_transmission` in the config file to the input `count` value.
- Modify the `setGigantoConfig` and `gigantoConfig` queries in the GraphQL API to read/write the ack transmission count.
- Modify the `setGigantoConfig` and `gigantoConfig` query in the GraphQL API so that the fields that take integers read/write the config file for their respective types.
- Modify the `gigantoConfig` query in the GraphQL API so that config files that work in standalone mode can also be read correctly.
- Changed `export` GraphQL API's response value format from `{export_path}` to `{export_path}@{giganto_node_name}`.
- Added `requestFromPeer` argument to GraphQL API: `netflow5RawEvents`, `netflow9RawEvents`, `secuLogRawEvents`,
  `statistics`.
- Added GraphQL API `sysmonEvents` to retrieve all sysmon events at once.
- The `secuLogRawEvents` GraphQL API requires its kind.
- Change the `statistics` GraphQL API.
- Add GraphQL API `statistics` to read data from `statistics` store.
- Added `search[sysmon type]Events` GraphQL APIs for sysmon event supported by giganto.
- Change `export` GraphQL API to support `statistics` store.
- Modify to use `Networkfilter` in GraphQL APIs requesting sysmon events.
- Fixed `export` GraphQL API to export statistics data for all cores.
- Fixed fields of `FtpRawEvent` and `FtpJsonOutput`.
- Added migration functionality in `0.12.0`. This feature adds values for new fields (`orig_filenames`, `orig_mime_types`, `resp_filenames`, `resp_mime_types`) to
  `Http` in versions 0.12.0 and earlier.
- Add a GraphQL API for `searchHttpRawEvents`.
- Add more fields to `Dns`, `Conn`, `Http`.

**Comment by sophie-cluml** [comment]
@jinjoolee07 I think this line needs update too. Left is main and right is current PR.
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
     This command must be applied automatically or manually before all future pull
     requests are submitted.
   - Add `--config group_imports=StdExternalCrate` to the CI process like:
-    - `cargo fmt -- --check --config group_imports=StdExternalCrate`
+    - `cargo fmt -- --check --config group_imports=StdExternalCrate`.
 - Modified network event search queries. The modifications are as follows.
   - Changed the type of filter from `NetworkFilter` to `Option<NetworkOptFilter>`.
-  - Added `request_from_peer` as a parameter to the queries.
+  - Added `requestFromPeer` as a parameter to the GraphQL API.
```
Comment (sophie-cluml) [inline CHANGELOG.md:22]:
```suggestion
  - Added `requestFromPeer` as a parameter to the GraphQL APIs.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   `source` to `timestamp`, `source`.
 - Removed `unsafe` block in `write_run_tcpdump` while creating a temorary file.
 - Modified `events_in_cluster` macro.
-  - Added `filter_into` expression to also support `total_count` query.
+  - Added `filter_into` expression to also support `totalCount` GraphQL API.
```
Comment (sophie-cluml) [inline CHANGELOG.md:32]: I think this line should kept as original.

**Review by sophie-cluml — APPROVED** [review]

**Pull request merged**